test/rtl/svEmitErrors.mlir

```diff
@@ -1,16 +1,13 @@
-// RUN: circt-translate %s -emit-verilog -verify-diagnostics --split-input-file
```

Comment:
Should we keep the `-emit-verilog` flag in sync with the nomenclature?

Reply:
Yeah! I'll be opening a PR after for that part